src/components/toolbox/flashMessage/flashMessage.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Icon from '../icon';
+import { SecondaryButtonV2 } from '../buttons/button';
+import styles from './flashMessage.css';
+
+const FlashMessage = ({
+  buttonClassName,
+  buttonText,
+  className,
+  displayText,
+  iconName,
+  linkCaption,
+  linkClassName,
+  linkUrl,
+  onButtonClick,
+}) => (
+  <div className={`${styles.wrapper} ${className}`}>
+    {
+      iconName
+        ? <Icon name={iconName} className="icon" />
+        : null
+    }
+    <span className={`${styles.text} display-text`}>
+      {displayText}
+    </span>
+    {
+      linkUrl && linkCaption
+        ? (
+          <a
+            className={`${styles.externalLink} ${linkClassName} url-link`}
+            href={linkUrl}
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            {linkCaption}
+          </a>
+        )
+        : null
+    }
+    {
+      buttonText
+        ? (
+          <SecondaryButtonV2
+            className={`${styles.button} ${buttonClassName} button`}
+            onClick={onButtonClick}
+          >
+            {buttonText}
+          </SecondaryButtonV2>
+        )
+        : null
+    }
+  </div>
+);
+
+FlashMessage.defaultProps = {
```

Comment:
Since  you are using the defaultProps, should be using also the propTypes, to define the type of each props, and also if it's required or not, required props, doesn't need a default value.